runtimes/pythonrt/config.go

```diff
@@ -13,17 +13,24 @@ type Config struct {
 	LogBuildCode      bool `koanf:"log_build_code"`
 
 	RunnerType string `koanf:"runner_type"`
+
+	CPUPerContainer      float32 `koanf:"cpu_per_container"`
+	MemoryPerContainerMB uint32  `koanf:"memory_per_container_mb"`
 }
 
 var Configs = configset.Set[Config]{
 	Default: &Config{
-		RunnerType: "local",
+		RunnerType:           "local",
+		CPUPerContainer:      1.0,
+		MemoryPerContainerMB: 128,
 	},
 	Test: &Config{
 		LazyLoadLocalVEnv: true,
 	},
 	Dev: &Config{
-		LogRunnerCode: true,
-		LogBuildCode:  true,
+		LogRunnerCode:        true,
+		LogBuildCode:         true,
+		CPUPerContainer:      1.0,
+		MemoryPerContainerMB: 128,
```

Comment:
How did we get to these values?

Reply:
also should  be configurable somehow

Reply:
or passed via arguments to the runtime or whatever

Reply:
they are configurable by env variables like any other config

got to the by trial and error to the minimal value I currently can work with

hopefully I can get it to 0.5 CPU and 64 MB memory